Automattic/woocommerce-payments
# Send current user fingerprint data on onboarding init

Fixes #7342 

#### Changes proposed in this Pull Request

We send data about the current user doing the onboarding as part of our efforts to improve our fraud prevention systems.

#### Testing instructions

* Checkout the PR branch on your local client installation
* Make sure you have the WCPay Dev Tools active so you are in dev mode (and the WooPayments logging is active)
* Onboard a new account
* Check the WooPayments WooCommerce [logs](http://localhost:8082/wp-admin/admin.php?page=wc-status&tab=logs) and you should see a `INFO REQUEST POST https://public-api.wordpress.com/wpcom/v2/sites/%s/wcpay/onboarding/init` with BODY data that should contain a `user_data` entry similar to this:

```
'user_data' => 
  array (
    'user_id' => 1,
    'ip_address' => '192.168.65.1',
    'browser' => 
    array (
      'user_agent' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 Safari/537.36',
      'accept_language' => 'en-GB,en-US;q=0.9,en;q=0.8,ro;q=0.7',
      'content_language' => 'en-US',
    ),
    'referer' => 'http://localhost:8082/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect',
  ),
```

-------------------


**Post merge**
